src/main/scala/ctrl.scala

```diff
@@ -79,23 +79,29 @@ class CtrlModule(val W: Int, val S: Int)(implicit p: Parameters) extends Module(
 
   val buffer_sram = p(BufferSram)
   //SRAM Buffer
-  val buffer_mem = Mem(UInt(width = W), round_size_words, seqRead = true)
+  //val buffer_mem = Mem(UInt(width = W), round_size_words, seqRead = true)
```

Comment:
Hey Colin,

Thanks for your suggestion. Would you like me to re-commit by removing
those lines?

On Wed, Mar 16, 2016 at 3:18 PM, Colin Schmidt <notifications@github.com>
wrote:

> In src/main/scala/ctrl.scala
> <https://github.com/ucb-bar/rocc-template/pull/5#discussion_r56425976>:
>
> > @@ -79,23 +79,29 @@ class CtrlModule(val W: Int, val S: Int)(implicit p: Parameters) extends Module(
> >
> >    val buffer_sram = p(BufferSram)
> >    //SRAM Buffer
> > -  val buffer_mem = Mem(UInt(width = W), round_size_words, seqRead = true)
> > +  //val buffer_mem = Mem(UInt(width = W), round_size_words, seqRead = true)
>
> We don't need to keep around the commented out version. Git will handle
> that for us.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/ucb-bar/rocc-template/pull/5/files/d38890e2f5518438727d7f6239c7b644b193c66d#r56425976>
>